eve/main.yml

```diff
@@ -285,6 +286,144 @@ stages:
+      - ShellCommand: &verify_prev_iso_checksum
+          name: Verify %(prop:metalk8s_prev_version)s checksum
+          command: sha256sum -c SHA256SUM
+          haltOnFailure: true
+      - ShellCommand: &create_prev_iso_mountpoint
+          name: Create %(prop:metalk8s_prev_version)s mountpoint
+          command: >
+            sudo mkdir -p /srv/scality/metalk8s-%(prop:metalk8s_prev_version)s
+      - ShellCommand: &mount_prev_iso
+          name: Mount %(prop:metalk8s_prev_version)s image
+          command: >
+            sudo mount -o loop metalk8s.iso
+            /srv/scality/metalk8s-%(prop:metalk8s_prev_version)s
+          haltOnFailure: true
+      - ShellCommand: &get_iso
+          name: Get %(prop:metalk8s_version)s image
+          command: >
+            curl -s -XGET -o /tmp/metalk8s.iso
+            "%(prop:artifacts_private_url)s/metalk8s.iso"
+          haltOnFailure: true
+      - ShellCommand: &get_iso_checksum
+          name: Get %(prop:metalk8s_version)s checksum
+          command: >
+            curl -s -XGET -o /tmp/SHA256SUM
+            "%(prop:artifacts_private_url)s/SHA256SUM"
+          haltOnFailure: true
+      - ShellCommand: &verify_iso_checksum
+          name: Verify %(prop:metalk8s_version)s checksum
+          command: cd /tmp && sha256sum -c SHA256SUM
+          haltOnFailure: true
+      - ShellCommand: &create_iso_mountpoint
+          name: Create %(prop:metalk8s_version)s mountpoint
+          command: >
+            sudo mkdir -p /srv/scality/metalk8s-%(prop:metalk8s_version)s
+      - ShellCommand: &mount_iso
+          name: Mount %(prop:metalk8s_version)s image
+          command: >
+            sudo mount -o loop /tmp/metalk8s.iso
+            /srv/scality/metalk8s-%(prop:metalk8s_version)s
+          haltOnFailure: true
+      - ShellCommand:
+          name: Create bootstrap configuration file
+          command: |
+            sudo bash << EOF
+            mkdir -p /etc/metalk8s
+            cat > /etc/metalk8s/bootstrap.yaml << END
+            apiVersion: metalk8s.scality.com/v1alpha2
+            kind: BootstrapConfiguration
+            networks:
+              controlPlane: 10.100.0.0/16
+              workloadPlane: 10.100.0.0/16
+            ca:
+              minion: $(hostname)
+            apiServer:
+              host: $(ip route get 10.100.0.0 | awk '/10.100.0.0/{ print $6 }')
+            archives:
+              - /srv/scality/metalk8s-%(prop:metalk8s_prev_version)s
+            END
+            EOF
+      - ShellCommand:
+          name: Start %(prop:metalk8s_prev_version)s bootstrap
+          command: >
+            sudo bash
+            /srv/scality/metalk8s-%(prop:metalk8s_prev_version)s/bootstrap.sh
+            --verbose || true
+          haltOnFailure: true
+      - ShellCommand:
+          name: Run upgrade to %(prop:metalk8s_version)s
+          command: >
+            sudo bash
+            /srv/scality/metalk8s-%(prop:metalk8s_version)s/iso-manager.sh
```

Comment:
you need to use the previous version here as normaly the new version is not available at this point